### PR TITLE
[Feature][UI Next] Add front-end for Apache Zeppelin task plugin (#9597)

### LINKED ISSUE #9597: [Feature][UI Next][V1.0.0-Beta] Add front-end for Apache Zeppelin task plugin
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

* Currently we have basic backend integration with `Apache Zeppelin` notebook in `dolphin scheduler zeppelin task plugin`. However, we need to add the front-end for this plugin.
* Related: #9201

### Use case

* Zeppelin task plugin will have corresponding front-end UI.

### Related issues

* See `Description` section.

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by EricGao888**
## Purpose of the pull request
* Add front-end for Apache Zeppelin task plugin
* Fix minor bug of zeppelin server endpoint default value in `common.properties`
* This PR closes: #9597 

## Brief change log
* Already described above

image

## Verify this pull request

* Verified by manual test

### CODE REVIEW

**Comment by EricGao888** [comment]
I don't know how to get a colorless icon of Apache Zeppelin. Do you have any ideas about how to do this? Thx. @songjianet @Amy0104 
image

**Comment by zhongjiajie** [comment]
what is going on, mergeable? 
image

**Comment by EricGao888** [comment]
> what is going on, mergeable? image

Looks like something wrong with the `mergeable bot`? LOL

**Comment by songjianet** [comment]
image

**Comment by EricGao888** [comment]
> image

Thx, I will fix it.

**Comment by Amy0104** [comment]
> I don't know how to get a colorless icon of Apache Zeppelin. Do you have any ideas about how to do this? Thx. @songjianet @Amy0104 image



> 
There are two ways to do this. Firstly, you can try photoshop to modify the icon picture. Secondly, you can use the  the filter in css style. But there is better to use the icon in svg format.

**Comment by EricGao888** [comment]
> > I don't know how to get a colorless icon of Apache Zeppelin. Do you have any ideas about how to do this? Thx. @songjianet @Amy0104 image
> 
> > 
> 
> There are two ways to do this. Firstly, you can try photoshop to modify the icon picture. Secondly, you can use the the filter in css style. But there is better to use the icon in svg format.

Thx, I will try the approaches you mentioned above : )

File: dolphinscheduler-ui-next/src/views/projects/task/components/node/tasks/use-zeppelin.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { reactive } from 'vue'
+import * as Fields from '../fields/index'
+import type { IJsonItem, INodeData } from '../types'
+import { ITaskData } from '../types'
```
Comment (Amy0104) [inline dolphinscheduler-ui-next/src/views/projects/task/components/node/tasks/use-zeppelin.ts]: Import the ITaskData together with IJsonItem and INodeData.

File: dolphinscheduler-ui-next/src/views/projects/task/constants/task-type.ts [hunk 2]
```diff
@@ -93,5 +94,8 @@ export const TASK_TYPES_MAP = {
   EMR: {
     alias: 'AmazonEMR',
     helperLinkDisable: true
+  },
+  ZEPPELIN: {
+    alias: 'ZEPPELIN'
```
Comment (Amy0104) [inline dolphinscheduler-ui-next/src/views/projects/task/constants/task-type.ts]: Is there a instruction for the new task type. If not, should set the helperLinkDisable.

File: dolphinscheduler-ui-next/src/views/projects/task/components/node/tasks/use-zeppelin.ts (hunk 1 above)
Comment (songjianet) [reply dolphinscheduler-ui-next/src/views/projects/task/components/node/tasks/use-zeppelin.ts]: Type imports require the use of `import type` .

Comment (EricGao888) [reply dolphinscheduler-ui-next/src/views/projects/task/components/node/tasks/use-zeppelin.ts]: Does it mean I should change it to `import type { IJsonItem, INodeData, ITaskData } from '../types'` ?

File: dolphinscheduler-ui-next/src/views/projects/task/constants/task-type.ts (hunk 2 above)
Comment (EricGao888) [reply dolphinscheduler-ui-next/src/views/projects/task/constants/task-type.ts]: No instructions yet, will set the helperLinkDisable. Thx

**Comment by EricGao888** [comment]
@songjianet @Amy0104 I had installed `vue-tsc` through cmd `npm i vue-tsc -D`. When I tried `vue-tsc`, I got error `zsh: command not found: vue-tsc`. Did I miss anything? Thx

**Comment by EricGao888** [comment]
CI seems good except the `SonarCloud Code Analysis`. Do we need to handle this analysis failure? @songjianet @Amy0104

**Comment by songjianet** [comment]
> CI seems good except the `SonarCloud Code Analysis`. Do we need to handle this analysis failure? @songjianet @Amy0104

unnecessary

File: dolphinscheduler-ui-next/src/views/projects/task/components/node/tasks/use-zeppelin.ts (hunk 1 above)
Comment (songjianet) [reply dolphinscheduler-ui-next/src/views/projects/task/components/node/tasks/use-zeppelin.ts]: Yes

File: dolphinscheduler-ui-next/src/views/projects/task/components/node/tasks/use-zeppelin.ts [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  data
+}: {
+  projectCode: number
+  from?: number
+  readonly?: boolean
+  data?: ITaskData
+}) {
+  const model = reactive({
+    name: '',
+    taskType: 'EMR',
```
Comment (Amy0104) [inline dolphinscheduler-ui-next/src/views/projects/task/components/node/tasks/use-zeppelin.ts]: The taskType should be 'ZEPPELIN' here.

Comment (EricGao888) [reply dolphinscheduler-ui-next/src/views/projects/task/components/node/tasks/use-zeppelin.ts]: Can't believe I missed this one, lol. Thx for pointing out! Will fix in the next commit : )

Comment (EricGao888) [reply dolphinscheduler-ui-next/src/views/projects/task/components/node/tasks/use-zeppelin.ts]: @Amy0104  Guess you all know I copied the code from `EMR task plugin` this time. :sweat_smile:

**Review by Amy0104 — APPROVED** [review]
LGTM.

**Review by songjianet — APPROVED** [review]
LGTM

**Pull request merged**